database/tests/conftest.py

```diff
@@ -84,9 +84,14 @@ def heart_rate():
 	return str(find_one_biometric(db)['heart_rate'])
 
 @pytest.fixture
-def blood_pressure():
+def systolic_blood_pressure():
```

Comment:
We don't need these blood pressure fixtures anymore

Reply:
Made the required changes.

Reply:
thanks, I like that a lot better.